nessie/hooks_test.go

```diff
@@ -83,6 +98,17 @@ func TestHooksSuccess(t *testing.T) {
 	uploadResp, err := uploadContent(ctx, repo, branch, "_lakefs_actions/testing_pre_commit", preCommitAction)
 	require.NoError(t, err)
 	require.Equal(t, http.StatusCreated, uploadResp.StatusCode())
+
+	actionPostCommitTmpl := template.Must(template.New("action-post-commit").Parse(actionPostCommitYaml))
```

Comment:
now that we got to the point where we copy this one for the third time, we can start reuse and extract to a func.